cmd/ak/common/flags.go

```diff
@@ -68,3 +69,19 @@ func ToExitCodeWithSkipNotFoundFlag(cmd *cobra.Command, err error, whats ...stri
 	}
 	return exitErr
 }
+
+// maybe we could just run this check for all required flags instead of providing explicit ones?
+func UnsetEmptyFlags(cmd *cobra.Command, flags ...string) {
+	flagsMap := make(map[string]struct{}, len(flags))
```

Comment:
it's easier just to `make(map[string]bool, len(flags)`. then check if `if flagsMap[whatever] {`.

Reply:
sorry, I failed to understand your comment.
I want to convert arguments passed to map to check them easily.
later in `Visit` there is a callback which is called for which flag set by user